lgc/patch/PatchLoopMetadata.cpp

```diff
@@ -86,7 +129,7 @@ PatchLoopMetadata::PatchLoopMetadata()
 //
 // @param [in,out] module : LLVM module to be run on
 bool PatchLoopMetadata::runOnModule(Module &module) {
-  LLVM_DEBUG(dbgs() << "Run the pass patch-loop-metadata\n");
+  LLVM_DEBUG(dbgs() << "Run the pass llpc-patch-loop-metadata\n");
```

Comment:
Remove "llpc-". The patch passes are in LGC :)

Reply:
Good point! I added the llpc- to match the DEBUG_TYPE without thinking about it - the llpc was from the earlier refactoring which moved the pass from llpc to lgc. I've changed llpc- to lgc- for both.